src/extensions/default/bramble/lib/ConsoleManagerRemote.js

```diff
@@ -0,0 +1,64 @@
+(function(transport) {
+    "use strict";
+
+    //TODO: add support for console.log(one,two, three) arguments
+
+    // Returns a function which calls the specific function in scope
+    var bind = function(func, scope, args) {
```

Comment:
This is already a function in JavaScript, what does this do differently from the regular `bind()`? And then specifically: can that be documented in the code here, because that's the important information that should be in the function documentation.

That said, is `bind` used anywhere? I don't see it referred to anywhere else in this file.